app/mailers/notification.rb

```diff
@@ -0,0 +1,4 @@
+class Notification < ActionMailer::Base
+  default :from => "robot@theoj.org"
```

Comment:
Fair enough. I try to keep to the old one :smile:

I'll update the PR to be consistent with your style.